src/main/java/com/microsoft/azure/servicebus/jms/AadAuthentication.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.servicebus.jms;
+
+import com.azure.core.credential.AccessToken;
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.credential.TokenRequestContext;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+
+public class AadAuthentication {
+
+    private final String AUDIENCE = "https://servicebus.azure.net/.default";
+    private TokenCredential credential;
+    private AccessToken currentAccessToken;
+    
+	public AadAuthentication() {
+		this(new DefaultAzureCredentialBuilder().build());
+	}
```

Comment:
Can this be moved to a different class altogether? Because if there is for ex, a different constructor of DefaultAzureCredentialBuilder that customer wishes to use, they will not be able to use it if we use it this way?

Reply:
May be remove this.

Reply:
It has been remove